cmd/depprobcheck/go.mod

```diff
@@ -3,62 +3,66 @@ module sigs.k8s.io/kustomize/cmd/depprobcheck
 go 1.19
 
 require (
-	k8s.io/cli-runtime v0.20.4
-	k8s.io/kube-openapi v0.0.0-20220401212409-b28bf2818661
+	k8s.io/cli-runtime v0.26.0
+	k8s.io/kube-openapi v0.0.0-20230109183929-3758b55a6596
```

Comment:
This and some other modules have various unrelated dependency bumps. Accidental? Dependency management is something we need to do really carefully because of our integration with the k/k repo, so we generally want each bump to go in its own PR for separate consideration.

Reply:
My mistake! Still figuring out go dependencies, I accidentally ran `go get -u` and didn't realize it would update all the things. I think I sorted it out but we can chat about it later today.

Reply:
Hi @KnVerey ,
I updated dependabot config to provide update per dependency and module in #4971 if you want to review this ?